### ISSUE TITLE
Nickname ranks.

### ISSUE THREAD

**Issue by flxrr** [McPlayHDnet/McPlayHD.net#134]
I think it'd be a good idea to change the default rank for a nickname back to the default player rank, unless you're nicked in fb01. As of now, many players that have played the server for a couple hours, and aren't that new to the server, can easily tell if someone is nicked. Most are aware that a nicked player has to be premium. Meaning if there's only 1 or 2 players with the premium rank in their lobby they would just have to check those players statistics in a different gamemode than what the nicked player is currently playing. If the stats are basically empty (0 attempts, etc) they would know that player is in fact, nicked.

tldr: make nicked players premium in fb01 and nons in other fbs
